# Paired Analysis of Lung Cancer Tumor Cells Using JIVE

This analysis addresses challenges in studying squamous cell lung carcinoma (LUSC) using The Cancer Genome Atlas (TCGA) data. TCGA, a major cancer genomics program, offers comprehensive multi-omics data, including for LUSC, a leading cause of cancer-related deaths globally. LUSC, often linked to smoking, exhibits genomic alterations distinct from other lung cancers. Key analytical challenges include the non-independence of normal and tumor tissue samples from the same patient, the high-dimensionality of the data (RNA-sequencing data with 60,660 genes and methylation data with 422,357 sites), and the integration of multiple data sources. To address these, the analysis employs Principal Component Analysis (PCA) for dimension reduction, using an eigendecomposition approach (Î£ğ‘Ã—ğ‘ = ğ‘ˆğ‘Ã—ğ‘ğ·ğ‘Ã—ğ‘ğ‘ˆğ‘ğ‘‡Ã—ğ‘) to approximate the data matrix (ğ‘‹ğ‘Ã—ğ‘› â‰ˆ ğ‘ˆğ‘Ã—ğ‘Ÿğ‘†ğ‘ŸÃ—ğ‘›). PCA's orthogonality helps handle multicollinearity. Further, to manage multi-source data integration, Joint and Individual Variation Explained (JIVE) extends PCA, enabling simultaneous analysis of high-dimensional data from various sources (ğ‘‹ğ‘˜ = ğ‘ˆğ‘˜ğ‘† + ğ‘Šğ‘˜ğ‘†ğ‘˜ + ğ‘…ğ‘˜). JIVE distinguishes between joint and individual variation across data sources, ensuring unique variation capture. Our methodology differs from traditional differential gene expression analysis by focusing on the significance of principal components from paired data sources, rather than comparing independent groups for each gene. 
